internal/util/util.go

```diff
@@ -22,3 +29,35 @@ func LookupImage(ctx *types.SystemContext, store storage.Store, image string) (*
 	}
 	return localImage, nil
 }
+
+// ExportFromReader reads bytes from given reader and exports to external tar, directory or stdout.
+func ExportFromReader(input io.Reader, isDir bool, isStdout bool, path string) error {
+	var err error
+	if !filepath.IsAbs(path) {
+		cwd, err := os.Getwd()
+		if err != nil {
+			return err
+		}
+		path = filepath.Join(cwd, filepath.Clean(path))
```

Comment:
`filepath.Join` implicitly runs `Clean` on the _output_. Does calling `Clean` on the _input_ make any non-obvious difference? If so, please document that.